peewee.py

```diff
@@ -1586,6 +1601,8 @@ def get_parse_map(self):
         }
 
     def quote(self, s):
+        if re.match(r'[a-zA-Z]\w*$', s) and s.upper() not in self.sql_reserved_words:
+          return s
```

Comment:
if it does it'll be compiling the `re`, which can be pulled out into a constant.  but my instinct says talking over the socket will dwarf it.  agreed on profiling.  do you already have a script set up here?